docs/spec-v1alpha1.md

```diff
@@ -0,0 +1,221 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
+
+- [OCI Model Configuration Specification](#oci-model-configuration-specification)
+  - [Terminology](#terminology)
+    - [Tensor](#tensor)
+    - [GitRepo](#gitrepo)
+  - [Properties](#properties)
+  - [Example](#example)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+# OCI Model Configuration Specification
+
+An OCI Model is a Machine Learning/Deep Learning model. This specification outlines the JSON format describing models for use and execution tool and its relationship to filesystem changesets.
+
+This section defines the `application/vnd.caicloud.model.config.v1alpha1+json` media type.
```

Comment:
Should we get rid of `caicloud` from the beginning?

Reply:
Then will we use kubeflow? Or a new org name?

Reply:
I think we should have a new org. Let's change this after we create the new org.